src/plugins/elasticsearch/lib/create_proxy.js

```diff
@@ -16,10 +16,11 @@ function createProxy(server, method, route, config) {
     handler: {
       proxy: {
         mapUri: mapUri(server),
-        passThrough: true,
         agent: createAgent(server),
-        xforward: true,
```

Comment:
I'm torn on this one. Its common for proxies to add `x-forwarded-*` headers and what we have here is indeed a proxy, so for that would be a reason to include these. On the other hand, `x-forwarded-for` will include information about the client (specifically, their IP address) and looking at comments such as https://github.com/elastic/kibana/issues/6221#issuecomment-183418042, I don't think we should include it.

I think, for now, we should err on the side of not including it. We can always add it in later if it is requested (either explicitly or via a config option) but it would be harder to take away later if we add it now and folks start relying on it somehow.